src/commands/users/update_user_token_name.ts

```diff
@@ -0,0 +1,110 @@
+import { GraphRbacManagementClient } from "@azure/graph";
+import Command from "@oclif/command";
+import * as Parser from "@oclif/parser";
+import chalk from "chalk";
+import cli from "cli-ux";
+import { toError } from "fp-ts/lib/Either";
+import { fromNullable } from "fp-ts/lib/Option";
+import { fromLeft, tryCatch } from "fp-ts/lib/TaskEither";
+// tslint:disable-next-line: no-submodule-imports
+import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
+import { getGraphRbacManagementClient } from "../../utils/azure";
+
+export class UserTokenNameUpdate extends Command {
+  public static description =
+    "Update the Token Name attribute associated to the User identified by the provided email";
+
+  // tslint:disable-next-line: readonly-array
+  public static args: Parser.args.IArg[] = [
+    {
+      description: "email",
+      name: "email",
+      required: true
+    },
+    {
+      description: "tokenNameValue",
+      name: "tokenNameValue",
+      required: true
+    }
+  ];
+
+  // tslint:disable-next-line: readonly-array
+  public static examples = [`$ io-ops users:update-token-name`];
+
+  public static flags = {};
+
+  public async run(): Promise<void> {
+    // tslint:disable-next-line: no-shadowed-variable
+    const { args } = this.parse(UserTokenNameUpdate);
+
+    cli.action.start(
+      chalk.blue.bold(
+        `Updating a token name for: ${args.email} setting it to ${args.tokenNameValue}`
+      ),
+      chalk.blueBright.bold("Running"),
+      {
+        stdout: true
+      }
+    );
+
+    const servicePrincipalCreds = {
+      clientId: getRequiredStringEnv("SERVICE_PRINCIPAL_CLIENT_ID"),
+      secret: getRequiredStringEnv("SERVICE_PRINCIPAL_SECRET"),
+      tenantId: getRequiredStringEnv("SERVICE_PRINCIPAL_TENANT_ID")
```

Comment:
that would be nice to login using the azure client or with username and password instead (so this command can be run without sharing the `SERVICE_PRINCIPAL_SECRET` of a sp with very high privileges).

Reply:
Added in [b920954](https://github.com/pagopa/io-ops/pull/128/commits/b920954548320f3fb5b39bc7e52b4e06910be43a)